study_assets_root/bistablePerception/mainTaskVideoStimuli.html

```diff
@@ -19,7 +19,7 @@
   <video id="bistableVideo" class="video-js vjs-default-skin" preload="auto" width="640" height="264" autoplay="true"
     loop="true" data-setup='{"example_option":true}'></video>
   <p id="feedbackKeyPress"></p>
-  <p id="quit">Quit Study</p>
+<!--  <p id="quit">Quit Study</p>-->
```

Comment:
Don't leave commented out code, it confuses the users

Reply:
sorry, I missed it :(. I will fix the same.